app/src/org/commcare/android/tasks/DumpTask.java

```diff
@@ -57,19 +56,15 @@
     public static final int BULK_DUMP_ID = 23456;
     
     DataSubmissionListener formSubmissionListener;
-    CommCarePlatform platform;
-    
-    SqlStorage<FormRecord> storage;

```

Comment:
A bunch of task classes have unused 'platform' and 'storage' fields. Taking them out.